src/tutorials/narratives-how-to-write.rst

```diff
@@ -211,7 +211,7 @@ The dataset URL :ref:`is detailed below<linking-view-to-url>` and defines the vi
 Linking the view into the data to the URL
 -----------------------------------------
 
-At the heart of narratives is the ability for URLs on nextstrain.org to encode not only the dataset to display but also the view settings, such as the coloring used, via the `URL query <https://en.wikipedia.org/wiki/Query_string>`__.  
```

Comment:
I'd like to keep the usage of "nextstrain.org URLs" to emphasize that these have to be auspice dataset URLs, not any URL on the web.  I've tried to write this tutorial as  a guide for writing narratives on nextstrain.org using the nextstrain.org/edit/narratives where using the nextstrain.org domain name, while not technically necessary, is a good idea. I'd advocate using the [Differences between localhost (Auspice) and nextstrain.org](https://github.com/nextstrain/docs.nextstrain.org/blob/3dcebe0c1eceb8853e06072dff94d789dd79f1b7/src/tutorials/narratives-how-to-write.rst#id38) section to convey the domain name details.

Reply:
What about "dataset URLs" or "Nextstrain dataset URLs"? This helps users solidify their understanding of what a "dataset" is, and doesn't compromise the technical accuracy of the description.

Reply:
Changed to fefcbd63450bcfc51712953c87040c4ea8eeb9fc and included in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/cc9731916ba8372195f615d7649f18007f7beff6..9a54741ffdbeadc41b85454b3b7f15a85f89eeb2).